map/lanelet2_extension/CMakeLists.txt

```diff
@@ -66,51 +45,34 @@ add_library( lanelet2_extension_lib
   lib/utilities.cpp
   lib/visualization.cpp
 )
-
-add_dependencies(lanelet2_extension_lib
-  ${${PROJECT_NAME}_EXPORTED_TARGETS}
-  ${catkin_EXPORTED_TARGETS}
-)
-
 target_link_libraries(lanelet2_extension_lib
-  ${catkin_LIBRARIES}
   ${GeographicLib_LIBRARIES}
 )
 
-add_executable(lanelet2_extension_sample src/sample_code.cpp)
-add_dependencies(lanelet2_extension_sample ${${PROJECT_NAME}_EXPORTED_TARGETS} ${catkin_EXPORTED_TARGETS})
+ament_auto_add_library(lanelet2_extension_sample src/sample_code.cpp)
```

Comment:
The `lanelet2_extension_sample` is changed from executable to library. Is this intentional? Same for the autoware_lanelet2_validation.

Reply:
This was unintentional. Thanks for pointing out.